clar2wasm/src/wasm_generator.rs

```diff
@@ -1643,7 +1643,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.tx_sponsor")
+                            .by_name("tx_sponsor")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.